logs_intake/socket/writeToTLSSocket.js

```diff
@@ -0,0 +1,34 @@
+tls = require('tls');
+var host = "intake.logs.datadoghq.com"
+var ssl_port = 10516
+
+// callback for when secure connection established
+function connected(stream) {
```

Comment:
this function is beautifully simple. can we not make the python example as beautifully simple?

Reply:
The python code shows how to merge tags into the logs being sent and does some error checking.